lib/infrastructure/logging/logger.js

```diff
@@ -0,0 +1,17 @@
+const log = (level, message, data = {}) =>
+  console.log(
+    JSON.stringify({
+      level,
+      timestamp: new Date(Date.now()).toISOString(),
+      handler: process.env._HANDLER,
```

Comment:
This is an AWS environment variable, makes an assumption we are running in Lambda - but equally should just be `undefined` elsewhere so not going to break anything.